docs/source/_src_user_guide/state_propagation/environment_setup/thrust_refactor/thrust_refactor.rst

```diff
@@ -0,0 +1,168 @@
+==================================================
+Thrust and Aerodynamic Orientation - Code Refactor
+==================================================
+
+.. toctree::
+   :titlesonly:
+   :hidden:
+   :maxdepth: 1
+
+   thrust_old
+   
+The framework in Tudat to calculate inertial thrust direction and aerodynamic guidance was overhaulen in mid-2022. The reasons for this were:
+
+* Uninformative error warnings for certain models (primarily: custom thrust directions)
+* Inconsistent ways in which a vehicle's orientation is defined (implicitly by both aerodynamic guidance and thrust guidance)
+* Unclear interaction between thrust and aerodynamics: both can define a body's orientation, so what happens if you use both?
+
+The refactor has addressed these issues, as well as a number of related points. Most of the changes in functionality have been made 'under the hood', and are not visble to users. However, two important aspects of the interfaces have been modified in a manner which is *not* backwards compatible. If a user accesses the 'old' function, they will be redirected to this page. Below is an overview of what has changes, why, and how to convert your old code to new code.
+
+Specifically, you are likely here because one of two pieces of code is no longer working: the definition of a thrust acceleration, or the definition of aerodynamic guidance. Below, we describe how to modify your old code to the new setup of Tudat, while retaining the identical functionality.
+
+Main modification - vehicle rotation
+====================================
+
+The driver behind the lack of backwards compatibility is the following new paradigm:
+
+* **All** methods in which to define a vehicle's orientation (with the exception of the numerical propagation of the vehicle's rotational dynamics) have to happen by *explicitly* setting the vehicle's rotation model. This means that 
```

Comment:
Maybe this is a nice place to add a few lines about the underlying motivation for such changes? You mention in the introduction that this page should also describe _why_ the thrust functionalities have undergone such refactoring but I'm not sure if this is actually done (maybe I overlooked it though). Anyway, probably no need to add a detailed explanation, just a few sentences should suffice(?)

Reply:
Good point, done :)